src/main/resources/db/migration/V1_50_1__contract_type_data.sql

```diff
@@ -0,0 +1,19 @@
+insert into contract_type(id, name, code)
+Values  ('72e60faf-b8e5-4699-9d7c-aef631cca71b', 'Accomodation', 'ACC'),
+        ('b402486d-991e-4977-9291-073a3526d60f', 'Education, Training and Employment', 'ETE'),
+        ('bf68c1c6-f83b-45ed-a6f7-495456e5745a', 'Finance, Debt and Benefits', 'RDB'), --why R?
+        ('e46e76e1-38da-46a4-afc0-d5e7283ebb03', 'Dependency and recovery', 'D&R'),
+        ('f9b59d2c-c60b-4eb0-8469-04c975d2e2ee', 'Personal Wellbeing', 'PWB'),
+        ('ab0ddafa-619a-47c1-89ca-3709b2833761', 'Lifestyle, associates and social inclusion', 'LSI');
```

Comment:
For now, it's enough to have:
```
Accommodation
Education, Training and Employment
Personal Wellbeing All ages
Personal Wellbeing 26+
Personal Wellbeing Young Adults
Women's Services
```

Reply:
Source: https://github.com/ministryofjustice/hmpps-interventions-seed-data/blob/main/real-data/contracts.csv

Reply:
And what service categories these actually map to?:
Personal Wellbeing All ages
Personal Wellbeing 26+
Personal Wellbeing Young Adults
Women's Services